### PR TITLE
throw clear exception in lookuptable

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

throw clear exception in lookuptable
fix https://github.com/intel-analytics/BigDL/issues/1583

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LookupTable.scala [hunk 1]
```diff
@@ -168,12 +168,18 @@ class LookupTable[T: ClassTag]
...
+    try {
+      if (inputBuffer.dim() == 1) {
+        output.index(1, inputBuffer, weight)
+      } else if (inputBuffer.dim() == 2) {
+        output.index(1, inputBuffer.view(inputBuffer.nElement()), weight)
+        output = output.view(inputBuffer.size(1), inputBuffer.size(2), weight.size(2))
+      }
+    } catch {
+      case e: Exception =>
+        throw new RuntimeException(s"LookupTable updateOutput get exception : ${e}")
```
Comment (qiuxin2012) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LookupTable.scala]: just `throw e` ?

**Comment by zhangxiaoli73** [comment]
Now, exception message is:
com.intel.analytics.bigdl.utils.LayerException was thrown.
Layer info: LookupTable[c42ea9d4](nIndex=10,nOutput=3,paddingValue=0.0,normType=2.0)
java.lang.IllegalArgumentException: LookupTable updateOutput get exception:requirement failed: 11 out of range 0 to 9
please ensure elements of your input will not exceed 10

**Comment by yiheng** [comment]
Can we merge the PR?

**Comment by zhangxiaoli73** [comment]
jenkins pass

**Pull request merged**